### PR TITLE
Zallen/raster colormap renderer

### PR DESCRIPTION
**Pull request by ZackAllen**
Initial implementation in WPF. Also changed readme to fit .NET implementation

### CODE REVIEW

File: src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/RasterColormapRenderer/readme.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+## Features
+- `Map`
+- `Basemap`
+- `ColormapRenderer`
+- `MapView`
+- `Raster`
+- `RasterLayer`
+
+## Offline Data
+Read more about how to set up the sample's offline data [here](http://links.esri.com/ArcGISRuntimeQtSamples).
```
Comment (nCastle1) [inline src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/RasterColormapRenderer/readme.md]: You'll want to be careful with the offline data sections, these almost always need to be changed from the common readme content. I also recommend getting rid of the table below and using a plain link, to ensure it renders properly everywhere and looks good in plain text.

File: src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/RasterColormapRenderer/readme.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- Create an `IEnumerable<Color>` and use it as a colormap: colors at the beginning of the list replace the darkest values in the raster and colors at the end of the list replaced the brightest values of the raster.
+- Create a colormap renderer with the colormap and apply it to the raster layer with `rasterLayer.Renderer = colormapRenderer`.
+
+## Features
+- `Map`
+- `Basemap`
+- `ColormapRenderer`
+- `MapView`
+- `Raster`
+- `RasterLayer`
```
Comment (nCastle1) [inline src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/RasterColormapRenderer/readme.md]: There are a few aspects of this readme that appear to be out of date. I recommend copying from the latest version to correct some formatting issues (super picky markdown stuff)

File: src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/RasterColormapRenderer/RasterColormapRenderer.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+using System.Drawing;
+using System.Linq;
+using System.Windows;
+
+namespace ArcGISRuntime.WPF.Samples.RasterColormapRenderer
+{
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Raster colormap renderer",
+        "Layers",
+        "Demonstrates how to use a colormap renderer on raster layer.",
```
Comment (nCastle1) [inline src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/RasterColormapRenderer/RasterColormapRenderer.xaml.cs]: You'll want to update this to match the common readme text, **Apply a colormap renderer to a raster.**

**Comment by ZackAllen** [comment]
Now on all platforms. Screenshots have been taken and readmes have been updated.

**Pull request merged**